.github/workflows/builds-and-tests.yml

```diff
@@ -142,6 +143,11 @@ jobs:
       if: matrix.mpi == 'par' && matrix.os == 'windows-2022'
       uses: mpi4py/setup-mpi@v1.0.3
 
+    - name: setup Windows-specific environment vars
+      if: matrix.os == 'windows-2022'
+      run: |
+        echo 'HYPRE_BUILD_SYSTEM=cmake' >> ${Env:GITHUB_ENV}
+
```

Comment:
I would have preferred the duplication of `get hypre` into `get hypre (Windows)` and `get hypre`, to be consistent with the rest.